### ISSUE TITLE
Crash ProcessStartInfo

### ISSUE THREAD

**Issue by HavokDoesThingsHere** [X-Hax/SA-Mod-Manager#151]
## Exception Report
```
SAMM Information:
	Manager Version: 1.2.9
	First Boot: True
	Current Game: Sonic Adventure 2
	Game Directory: C:\Program Files (x86)\Steam\steamapps\common\Sonic Adventure 2
	Manager Location: C:\Users\Ryan King\Downloads\sa2modloader_7eae5\
	Running as Admin: False
	Mod Loader Version (Hash): 16e963f9ef335dd521e115ef59f5a79ff6d957c4
```

```
Exception Information:
	Type: Win32Exception
	Exception: An error occurred trying to start process 'https://github.com/End-User-Person/SA2B-DC-Characters' with working directory 'C:\Users\Ryan King\Downloads\sa2modloader_7eae5'. The system cannot find the file specified.
	Source: System.Diagnostics.Process
	In Function: Boolean StartWithCreateProcess(System.Diagnostics.ProcessStartInfo)
	StackTrace:    at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
	   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
	   at SAModManager.ModsCommon.ModDependency.CheckDependencies(List`1 modlist, Dictionary`2 allmods)
	   at SAModManager.MainWindow.SaveAndPlayButton_Click(Object sender, RoutedEventArgs e)
	   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
	   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
	   at System.Windows.Controls.Primitives.ButtonBase.OnClick()
	   at System.Windows.Controls.Button.OnClick()
	   at System.Windows.Controls.Primitives.ButtonBase.OnMouseLeftButtonUp(MouseButtonEventArgs e)
	   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
	   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
	   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
	   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
	   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
	   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
	   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
	   at System.Windows.Input.InputManager.ProcessStagingArea()
	   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
	   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
	   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
	   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
	   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
	   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
	   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
	   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
	   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
	   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
	   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
	   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
	   at System.Windows.Application.RunDispatcher(Object ignore)
	   at System.Windows.Application.RunInternal(Window window)
	   at SAModManager.App.Main()
```

**Comment by ItsEasyActually** [comment]
Were you attempting to open a link for a dependency mod via the pop up?

**Comment by ItsEasyActually** [comment]
Fixed in [1ab7948](https://github.com/X-Hax/SA-Mod-Manager/commit/1ab79485ed55ef4c05d67d2cd99286eccfe8fd9a)
